test/integration/full/preload/preload.js

```diff
@@ -107,7 +107,7 @@ describe('axe.utils.preload integration test', function() {
 		});
 	});
 
-	it('returns NO preloaded CSSOM assets when requested stylesheet does not exist`', function(done) {
+	it.skip('returns NO preloaded CSSOM assets when requested stylesheet does not exist`', function(done) {
```

Comment:
Either remove the test of fix them. If we leave them in this state we might never going to turn them back on.